soc/riscv/riscv-ite/it8xxx2/Kconfig.soc

```diff
@@ -66,6 +66,19 @@ config SOC_IT8XXX2_CPU_IDLE_GATING
 	  gated by individual drivers. When this option is disabled, CPU idle
 	  mode is always permitted.
 
+config SOC_IT8XXX2_JTAG_ENABLE
+	bool "The JTAG module clock gating is configured here"
+	help
+	  This option determines if the JTAG module would be enabled or not.
```

Comment:
`Enable support  for JTAG module gating.`

Reply:
Got it.